ark/phenotyping/som_utils.py

```diff
@@ -117,13 +282,6 @@ def create_fov_pixel_data(fov, channels, img_data, seg_labels,
     seg_labels_flat = seg_labels.flatten()
     pixel_mat['segmentation_label'] = seg_labels_flat
 
-    # remove any rows that sum to 0
```

Comment:
If we remove this here, then we never frequency normalize the data for training the SOM

Reply:
Per lab meeting, instead of row and column normalizing in R, we'll move this functionality to `som_utils`.

Reply:
@ngreenwald on second thought, I think it's easier to do just the row normalizing in Python and the column normalizing in R. Reason being while row normalizing can happen per fov, column normalizing cannot. Since we need to load the full subsetted data in R already, best to do column normalization there (don't want to load it again in Python).